x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/indexlifecycle/TimeseriesLifecycleTypeTests.java

```diff
@@ -303,6 +304,10 @@ public void testGetNextActionName() {
         // Hot Phase
         assertNextActionName("hot", RolloverAction.NAME, null, new String[] {});
         assertNextActionName("hot", RolloverAction.NAME, null, new String[] { RolloverAction.NAME });
+        assertNextActionName("hot", UnfollowAction.NAME, null, new String[] {});
+        assertNextActionName("hot", UnfollowAction.NAME, null, new String[] {UnfollowAction.NAME});
+        assertNextActionName("hot", UnfollowAction.NAME, RolloverAction.NAME,
+            new String[] {UnfollowAction.NAME, RolloverAction.NAME});
```

Comment:
I did not update this test when I added Unfollow to the Warm and Cold phases - I'll do that later today.